specs/interop/derivation.md

```diff
@@ -34,26 +34,40 @@ executing messages.
 - An executing message MUST have a corresponding initiating message
 - The initiating message referenced in an executing message MUST come from a chain in its dependency set
 - A block MUST be considered invalid if it is built with any invalid executing messages
-- The timestamp of the identifier MUST be greater than the interop network upgrade timestamp
-- The timestamp of the identifier MUST be less than or equal to the timestamp of the block that includes it
-- The timestamp of the identifier MUST be greater than timestamp of the block that includes it minus the expiry window
+- The block number `n` is `>` the activation block number, see [Activation Block](#activation-block).
+- The timestamp `t` of the identifier MUST be:
+  - `t <= execution_timestamp`, where `execution_timestamp` is the timestamp of the block
+    that includes the executing message.
+  - `t > execution_timestamp - expiry_window`, where `expiry_window` is the expiry window in seconds.
 
 L2 blocks that produce invalid executing messages MUST not be allowed to be considered safe.
 They MAY optimistically exist as unsafe blocks for some period of time. An L2 block that is invalidated
 because it includes invalid executing messages MUST be replaced by a deposits only block at the same
 block height. This guarantees progression of the chain, ensuring that an infinite loop of processing
 the same block in the proof system is not possible.
 
-## Omit User Transactions in Fork Activation Block
+## Activation Block
```

Comment:
Specs doesn't have a do-not-merge label.
Adding this comment to block instead until @clabby has checked

Reply:
https://github.com/ethereum-optimism/specs/pull/694#issuecomment-2847971979